pkg/omf/cli/omf.packages.cd.fish

```diff
@@ -0,0 +1,22 @@
+# SYNOPSIS
+#   Change the current working directory to plugin / theme home directory.
+#
+# OPTIONS
+#   <name> Name of the package or theme.
+
+function omf.packages.cd -a name -d "Change the current working directory to plugin / theme home directory."
+  if test -z "$name"
+    pushd $OMF_PATH
+    return 0
+  end
+
+  for path in {$OMF_PATH,$OMF_CONFIG}/{pkg,themes}/$name
```

Comment:
This is actually a missing feature on `omf cd`... But I'm fine with having the custom one coming first.